pkg/ipcache/types/types.go

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package ipcachetypes
```

Comment:
I'll add a patch on top for this, the package name was already defined in another file so I can't just fix it up in this one file in the commit that adds this since they all need to be consistent :+1: